AzureCalling/app/src/main/AndroidManifest.xml

```diff
@@ -52,7 +52,7 @@
         <activity
             android:name="com.azure.samples.communication.calling.activities.CallActivity"
             android:exported="true"
-            android:configChanges="orientation|screenSize|screenLayout"
+            android:configChanges="screenSize|screenLayout"
```

Comment:
@phileo99 do we need need orientation here?

Reply:
@iaulakh you're right, we are implementing `onConfigurationChanged()`.  I have added that configChange back in